libbeat/processors/dissect/field.go

```diff
@@ -108,6 +108,8 @@ func (f skipField) IsSaveable() bool {
 // message: hello world
 // result:
 //	hello: world
+//
+// Deprecated: see pointerField
```

Comment:
Should we also log somewhere that it's deprecated if a user uses it?

Reply:
This won't break any user config, I will merge this and create a followup PR to add a warning.